### ISSUE TITLE
Wider output from processes resource

### ISSUE THREAD

**Issue by koka424** [inspec/inspec#4458]
## Detailed Description

While trying to check some flags that were set on a running process I found that the output of the `command` method was cut off:

```bash
inspec> processes('java').commands
=> ["java -D[Standalone] -server -Xms96m -Xmx512m -XX:MetaspaceSize=96M -XX:Max"]
```

This output does not contain the flags that I need to check.

My testing setup is on a Centos 7 docker image.

```bash
You are currently running on:

    Name:      centos
    Families:  redhat, linux, unix, os
    Release:   7.6.1810
    Arch:      x86_64
```

inspec version:

```bash
# inspec --version
4.16.0
```

## Context

Having the entire command would allow me to verify that all flags were passed to the process and applied correctly.

## Possible Implementation

The `process` resource is running 

```bash 
ps axo label,pid,pcpu,pmem,vsz,rss,tty,stat,start,time,user:32,command
```

Appending `ww` to that command prints out the full PID command:

```bash
inspec> processes('java').commands
=> ["java -D[Standalone] -server -Xms96m -Xmx512m -XX:MetaspaceSize=96M -XX:MaxMetaspaceSize=256m -Djava.net.preferIPv4Stack=true -Djboss.modules.system.pkgs=org.jboss.byteman -Djava.awt.headless=true -Djava.security.egd=file:/dev/./urandom -Dorg.jboss.boot.log.file=/opt/wildfly/standalone/log/server.log -Dlogging.configuration=file:/opt/wildfly/standalone/configuration/logging.properties -jar /opt/wildfly/jboss-modules.jar -mp /opt/wildfly/modules org.jboss.as.standalone -Djboss.home.dir=/opt/wildfly -Djboss.server.base.dir=/opt/wildfly/standalone -c standalone.xml -b 0.0.0.0 -P=/etc/wildfly/wildfly.properties"]
```

I asked on the inspec channel in the Chef Slack community and it was recommended that I reference this earlier issue:

https://github.com/inspec/inspec/issues/1497#issuecomment-281286748

If I can provide any other information please let me know.

Thank you!

**Comment by clintoncwolfe** [comment]
I believe we have a PR on this, #3700

**Comment by koka424** [comment]
I apologize. I guess I did not search very well. This can be closed as a duplicate. I do not know if the proper etiquette is for me to close this issue or let you guys do it. If you would like me to close I can do that.

Thank you for the response!

**Comment by zenspider** [comment]
Since we don't have an issue tracking #3700, I'd like to keep this live.

**Comment by clintoncwolfe** [comment]
It looks like this was fixed for linux on #5232, and cannot be safely fixed for other unix OS because of the variety of flags to ps. I'm going to close this.
